### PR TITLE
Benchmarks for DFP Production Pipeline

### PR DESCRIPTION
**Pull request by bsuryadevara**
- Benchmarks for DFP Production Pipeline (Stages vs Modules)
- Added README.md
- Added Config files

```
----------------------------------------------------------------------------------------------------- benchmark: 6 tests -----------------------------------------------------------------------------------------------------
Name (time in ms)                               Min                    Max                   Mean                StdDev                 Median                   IQR            Outliers     OPS            Rounds  Iterations
------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
test_dfp_inference_duo_stages_e2e          308.4402 (1.0)         441.9953 (1.0)         385.4835 (1.0)         52.7466 (1.0)         374.8979 (1.0)         73.1232 (1.0)           2;0  2.5941 (1.0)           5           1
test_dfp_inference_azure_stages_e2e        454.2198 (1.47)        625.3723 (1.41)        539.4551 (1.40)        77.5497 (1.47)        556.1858 (1.48)       143.2852 (1.96)          2;0  1.8537 (0.71)          5           1
test_dfp_training_duo_modules_e2e       13,701.4709 (44.42)    15,542.6684 (35.16)    14,604.7726 (37.89)      806.8470 (15.30)    14,486.1345 (38.64)    1,461.3735 (19.99)         2;0  0.0685 (0.03)          5           1
test_dfp_training_duo_stages_e2e        14,617.3350 (47.39)    15,589.4445 (35.27)    14,941.8147 (38.76)      403.5400 (7.65)     14,717.5218 (39.26)      526.5890 (7.20)          1;0  0.0669 (0.03)          5           1
test_dfp_training_azure_stages_e2e      26,091.4968 (84.59)    27,554.4906 (62.34)    27,014.1010 (70.08)      558.1178 (10.58)    27,148.0393 (72.41)      612.2293 (8.37)          1;0  0.0370 (0.01)          5           1
test_dfp_training_azure_modules_e2e     26,228.4464 (85.04)    29,457.1970 (66.65)    28,156.9607 (73.04)    1,252.0302 (23.74)    28,241.6172 (75.33)    1,698.1469 (23.22)         2;0  0.0355 (0.01)          5           1
------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
```

### CODE REVIEW

**Review by drobison00 — APPROVED** [review]
Looks good.

**Review by mdemoret-nv — APPROVED** [review]

File: examples/digital_fingerprinting/production/morpheus/benchmarks/README.md [hunk 1]
```diff
@@ -0,0 +1,127 @@
+<!--
```
Comment (dagardner-nv) [inline examples/digital_fingerprinting/production/morpheus/benchmarks/README.md:1]: Can you refer to this doc from within either `docs/source/developer_guide/guides/5_digital_fingerprinting.md` or `docs/source/developer_guide/guides/6_digital_fingerprinting_reference.md`

Comment (bsuryadevara) [reply examples/digital_fingerprinting/production/morpheus/benchmarks/README.md:1]: @dagardner-nv referred as suggested

**Review by dagardner-nv — APPROVED** [review]

**Pull request merged**